### PR TITLE
Accounts UI - Create Account/Login functionality 

### PR DESCRIPTION
**Pull request by AustinShouli**
Completed basic login and create account functionality (still needs input validation). Can now create a new account, which gets added to the database. user is returned to the login page, which displays a message on the status of the account creation (pass/fail) and allow the user to sign in with the new credentials. If the account is not found, the user can't log in and a message explains that the account was not found. 

Added basic CSS for index (login) page.

Cleaned up accounts.py code and worked on getAllCategories and getAllSubs functions. These are commented out, as the categories.py code will not currently run. These will be tested after categories.py is tested and working.

### CODE REVIEW

**Review by RuochiW — CHANGES_REQUESTED** [review]
Please remove all the caches

**Review by RuochiW — CHANGES_REQUESTED** [review]
please remove .gitignore file

**Review by RuochiW — CHANGES_REQUESTED** [review]
it have more errors than db error

File: src/accounts.py [hunk 1]
```diff
@@ -168,15 +173,15 @@ def createAccount(aType, uname, passw, fname, lname, email):
...
     AccountController.activeUser.setPassword(passw)
-    AccountController.activeUser.setFirstName(fname)
-    AccountController.activeUser.setLastName(lname)
+    AccountController.activeUser.setfirst_name(fname)
+    AccountController.activeUser.setlast_name(lname)
     AccountController.activeUser.setEmail(email)
 
     #then call writeDB to update in DB
-    if data.writeAccount(AccountController.activeUser):
+    if account_data_controller.write_account(AccountController.activeUser):
```
Comment (RuochiW) [inline src/accounts.py]: Do not use return as a bool, it is a list

File: src/accounts.py [hunk 2]
```diff
@@ -145,15 +150,15 @@ def __init__(self, activeUser):
...
```
Comment (RuochiW) [inline src/accounts.py]: Do not use return as a bool, it is a list

**Review by RuochiW — CHANGES_REQUESTED** [review]
change the way to call database

File: server.py [hunk 3]
```diff
@@ -17,33 +30,83 @@ def success(name):
...
+
+            print("Server Sees Logged In. accountID is: {}".format(loginAtempt[1]), file=sys.stdout)
             return render_template('logged_in.html')
-            #AcController.activeUser.accountID = loginAtempt[2]
-            
+
         else: 
-            return redirect(url_for('LoginFailed', name=user))
+  
+            print("DB Error: {}", loginAtempt[1], file=sys.stdout) #Debugging: sql error to terminal
```
Comment (RuochiW) [inline server.py]: error type is not only about database

**Review by RuochiW — CHANGES_REQUESTED** [review]
maybe "Error: " is better way to do it

File: server.py (hunk 3 above)
Comment (AustinShouli) [reply server.py]: The output from that statement is the exception returned from SQL when checking the login credentials in the database, so yes it is always a database error. Besides, that statement is only output to the terminal, not the application.

**Pull request closed without merging**